glide-core/src/protobuf/redis_request.proto

```diff
@@ -234,6 +234,10 @@ enum RequestType {
     Dump = 193;
     Restore = 194;
     SortReadOnly = 195;
+    XPending = 196;
+    XClaim = 197;
+    XAutoClaim = 198;
+    XInfo = 199;
```

Comment:
a mistake?

Reply:
no - I might as well push all of these

Reply:
Why? Those are out of scope, no?

Reply:
sure.  but somewhat related.

Reply:
lolwut
Can you clarify?

Reply:
I wanted to add all the stream commands to get it over with.  I can remove the redundant ones if you are strongly opposed.  But I'm coding it all at the same time and this will save CI checks in the future.